newrelic/resource_newrelic_app_label.go

```diff
@@ -0,0 +1,172 @@
+package newrelic
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/newrelic/newrelic-client-go/pkg/apm"
+	"github.com/newrelic/newrelic-client-go/pkg/errors"
+	"log"
+	"strings"
```

Comment:
We try to organize imports with core first, then white space, then third-parties for clarity:

```
import (
  "log"
  "strings"

  "github.com/hashicorp/terraform-plugin-sdk/helper/schema"
  "github.com/newrelic/newrelic-client-go/pkg/apm"
  "github.com/newrelic/newrelic-client-go/pkg/errors"
)
```

Reply:
Reordered as suggested